### PR TITLE
Fix the filer connection of cassandra

### PR DESCRIPTION
**Pull request by eshujiushiwo**
Filer can use a cassandra cluster as the storage of metadata.
After the fix we can use more than one cassandra node for filer.

Due to the function `NewCluster` in gocql, now we support two cassandra nodes setting only

### CODE REVIEW

**Comment by chrislusf** [comment]
weed/filer/cassandra_store/cassandra_store.go:35: cannot use hosts (type []string) as type string in argument to strings.Split
weed/filer/cassandra_store/cassandra_store.go:38: cannot convert hosts (type []string) to type string

**Comment by eshujiushiwo** [comment]
@chrislusf Fixed.

File: weed/filer/cassandra_store/cassandra_store.go [hunk 1]
```diff
@@ -30,9 +30,16 @@ type CassandraStore struct {
...
 }
 
-func NewCassandraStore(keyspace string, hosts ...string) (c *CassandraStore, err error) {
+func NewCassandraStore(keyspace string, hosts string) (c *CassandraStore, err error) {
 	c = &CassandraStore{}
-	c.cluster = gocql.NewCluster(hosts...)
+	        s := strings.Split(hosts, ",")
+        if len(s) == 1 {
+
+		glog.V(2).Info("Only one  cassandra node to connect!A Cluster is Proposed!Now using:", string(hosts))
```
Comment (chrislusf) [inline weed/filer/cassandra_store/cassandra_store.go:37]: remove the double empty spaces

Comment (chrislusf) [reply weed/filer/cassandra_store/cassandra_store.go:37]: change "Proposed" to "recommended".

File: weed/filer/cassandra_store/cassandra_store.go [hunk 2]
```diff
@@ -30,9 +30,16 @@ type CassandraStore struct {
...
+                c.cluster = gocql.NewCluster(hosts)
+        } else if len(s) > 1 {
+                c.cluster = gocql.NewCluster(s[0], s[1])
```
Comment (chrislusf) [inline weed/filer/cassandra_store/cassandra_store.go]: why not using all of the hosts?

**Comment by eshujiushiwo** [comment]
@chrislusf using all server now.

**Comment by chrislusf** [comment]
Thanks!

**Pull request merged**